src/ducks/transactions/helpers.js

```diff
@@ -123,8 +123,21 @@ export const isNew = transaction => {
   return parseInt(transaction._rev.split('-').shift(), 10) <= 2
 }
 
-export const getReimbursementStatus = transaction =>
-  transaction.reimbursementStatus || 'no-reimbursement'
+export const getReimbursementStatus = transaction => {
+  if (isHealthExpense(transaction)) {
+    return getHealthExpenseReimbursementStatus(transaction)
+  }
+
+  return transaction.reimbursementStatus || 'no-reimbursement'
+}
+
+export const getHealthExpenseReimbursementStatus = transaction => {
+  if (transaction.reimbursementStatus) {
+    return transaction.reimbursementStatus
+  }
+
+  return isFullyReimbursed(transaction) ? 'reimbursed' : 'pending'
+}
```

Comment:
At some point, having this helpers in a reimbursement.js file would be good, it could then be shared in doctypes.

Reply:
I don't understand what you mean by « it could then be shared in doctypes ». To me, this helper is very coupled to the `io.cozy.bank.operations` doctype.